documentation/modules/ROOT/pages/integrations/openlineage.adoc

```diff
@@ -0,0 +1,355 @@
+// Category: debezium-using
+// Type: assembly
+// ModuleID: open-lineage-integration
+// Title: OpenLineage Integration
+[id="open-linegage-integration"]
+= OpenLineage Integration
+
+:toc:
+:toc-placement: macro
+:linkattrs:
+:icons: font
+:source-highlighter: highlight.js
+
+toc::[]
+
+{prodname} provides built-in integration with OpenLineage to automatically track data lineage for Change Data Capture (CDC) operations. 
```

Comment:
@mfvitale Does it make sense to add OLNG support to sink connectors too? If yes then maybe we should think a bit more about docs structure as source and sink connectors will be a bit different. Then it probably makes sense to wite this part as explicitly source connector and have a sink part saying like this is not done yet.

Reply:
> Does it make sense to add OLNG support to sink connectors too?

I think for `sink` you are referring to JDBC and Mongo right? If yes, then your are right.